scripts/travis/start_elasticdl_job.sh

```diff
@@ -0,0 +1,56 @@
+JOB_TYPE=$1
+
+kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
+
+if [[ "$JOB_TYPE" == "odps" ]] && \
+{ [[ "$ODPS_ACCESS_ID" == "" ]] || \
+[[ "$ODPS_ACCESS_KEY" == "" ]]; }; then
+    echo "Skipping ODPS related integration tests since \
+    either ODPS_ACCESS_ID or ODPS_ACCESS_KEY is not set"
+else
+    echo "Running ElasticDL job: ${JOB_TYPE}"
+    if [[ "$JOB_TYPE" == "odps" ]]; then
+        export MAXCOMPUTE_TABLE="odps_integration_build_"`
+            `"${TRAVIS_BUILD_NUMBER}_$(date +%s)"
+        docker run --rm -it \
+            -e MAXCOMPUTE_TABLE=$MAXCOMPUTE_TABLE \
+            -e MAXCOMPUTE_PROJECT=$MAXCOMPUTE_PROJECT \
+            -e MAXCOMPUTE_AK=$ODPS_ACCESS_ID \
+            -e MAXCOMPUTE_SK=$ODPS_ACCESS_KEY \
+            -v $PWD:/work -w /work elasticdl:dev bash \
+            -c 'python -c '`
+            `'"from elasticdl.python.tests.test_utils import *;'`
+            `'create_iris_odps_table_from_env()"'
+    fi
+    PS_NUM=2
+    WORKER_NUM=1
+    docker run --rm -it --net=host \
+        -e MAXCOMPUTE_TABLE=$MAXCOMPUTE_TABLE \
+        -e MAXCOMPUTE_PROJECT=$MAXCOMPUTE_PROJECT \
+        -e MAXCOMPUTE_AK=$ODPS_ACCESS_ID \
+        -e MAXCOMPUTE_SK=$ODPS_ACCESS_KEY \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v $HOME/.kube:/root/.kube \
+        -v /home/$USER/.minikube/:/home/$USER/.minikube/ \
+        -v $(pwd):/work \
+        -w /work elasticdl:ci \
+        bash -c "scripts/client_test.sh \
+        ${JOB_TYPE} ${PS_NUM} ${WORKER_NUM}"
+    if [[ "$JOB_TYPE" != "local" ]]; then
```

Comment:
Do we still need type "local"? I'd thought the local mode was developed at the very beginning stage of prototyping and will no longer be used.  If so, how about we remove this test and even the local mode source code in the next PR?

Reply:
We developed "local" to test the training loop using the `dataset_fn` defined in the model file. But it is not user-friendly because users should execute it using ElasticDL package and an ElasticDL client command. There may be a better solution to do it. If users can use the `dataset_fn` to get a `tf.data.Dataset`,  they can use `model.fit(dataset)` to test the training loop like a Kera-native program locally.

So, I will remove it and source codes in the next PR. Then I will design the proposed solution to get a `tf.dataDataset` using `dataset_fn` defined in the model file.